runner/job_manager.py

```diff
@@ -15,6 +17,7 @@
 This module is responsible for creating, submitting and monitoring the pools and jobs
 
 """
+_keyVault_param_identifier = "KEYVAULT_"
 
```

Comment:
Can you move this to the utils.py file like the method below?

Reply:
Done